docs/guides/getting-started-orchestrate-human-tasks.md

```diff
@@ -206,29 +249,54 @@ If the properties panel for your task doesn't open automatically, navigate to **
 Forms linked in the user task are deployed together with the process. If you make changes to a form, you have to deploy the referencing process again to make the changes appear.
 :::
 
-## Step 4: Run your process
+<Tabs groupId="install" className="tabs-hidden">
+<TabItem value="saas">
 
-Your process is now ready to run. Given its human-centric nature, it is well suited to be run in Tasklist. In order to make it accessible from Tasklist, the process must be deployed first.
+## Step 4: Validate your process using Play
 
-:::tip
-Human-centric processes involving user tasks seamlessly unfold within Tasklist, offering a cost-effective orchestration solution for human work with forms. However, the versatility of these processes extends beyond Tasklist, encompassing various alternative methods and applications. For instance, users can be redirected to external applications to fulfill tasks, bespoke task applications can be developed for any domain, or interactions with the physical world can be captured through event signals from sensors and IoT devices.
-:::
+You can use [Play mode](/components/modeler/web-modeler/play-your-process.md) to quickly validate the process behavior and play different scenarios.
 
-### Deploy and test run
+1. Click the **Play** tab to enter Play mode.
+1. Once the Play environment is ready, click **Start a process instance** to start testing your process.
```

Comment:
With 8.6, the user will need a cluster rather than just waiting for the Play environment.
![image](https://github.com/user-attachments/assets/c9c659e3-9afb-41cf-b7d2-d2deaa640df8)
![image](https://github.com/user-attachments/assets/840778c7-7365-4a94-8c67-5e8575e44746)
![image](https://github.com/user-attachments/assets/145e1d21-4ffa-4cc2-933c-5ea63ac73083)
![image](https://github.com/user-attachments/assets/e6b10b33-6ea3-4834-83c6-d42433029b6c)